tailor-app/src/main/java/com/future/tailormade/tailor_app/core/model/ui/order/OrderUiModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.future.tailormade.tailor_app.core.model.ui.order
+
+data class OrderUiModel(
+
+    var id: String,
+
+    var orderDate: String,
+
+    var design: OrderDesignUiModel,
+
+    var quantity: String,
+
+    var totalDiscount: String,
```

Comment:
how about total discount and price in double?

Reply:
This value has been mapped to String in UiModel to set the text directly kak
Any concern kak?

Reply:
Has been resolved